app/controllers/events_controller.rb

```diff
@@ -0,0 +1,92 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: [:show, :edit, :destroy, :attend]
+  #before_action :correct_user, only: %i[edit update]
+
+  # GET /events
+  # GET /events.json
+  def index
+    if params[:events].nil? || params[:events][:type] == 'upcoming'
+      @events = Event.upcoming.paginate(page: params[:page])
+      @header = 'Upcoming Events'
+    else
+      @events = Event.prev.paginate(page: params[:page])
+      @header = 'Previous Events'
+    end
+  end
+
+  # GET /events/1
+  # GET /events/1.json
+  def show
+    @event = Event.find(params[:id])
+    @event_attendees = @event.attendees
+    #@going = @event.attendees.where('accepted AND NOT declined')
+    #@not_going = @event.attendees.where('NOT accepted AND declined')
+  end
+
+  # GET /events/new
+  def new
+    @event = Event.new
+  end
+
+  # GET /events/1/edit
+  def edit
+    session[:event_id] = params[:id]
+    @event = Event.find(params[:id])
+    @users = User.where('id != ?', current_user.id)
+  end
+
+  # POST /events
+  # POST /events.json
+  def create
+    @event = Event.new(event_params.merge(user_id: current_user.id))
```

Comment:
Guys, there is a simple way to do this using the `build` method generated by the relation between users and events. Please, change this line to use that method.